src/components/views/elements/Field.js

```diff
@@ -21,15 +21,16 @@ export default class Field extends React.PureComponent {
     static propTypes = {
         // The field's ID, which binds the input and label together.
         id: PropTypes.string.isRequired,
-        // The field's <input> type. Defaults to "text".
+        // The element to create. Defaults to "input".
+        // Should be "input", "select", or "textarea".
+        // To define options for a select, use <Field><option ... /></Field>
+        element: PropTypes.string,
```

Comment:
If it can only be one of three strings maybe using oneOf is beneficial here

Reply:
Thanks, good idea, added!